app/assets/javascripts/views/file_uploader_view.js.coffee

```diff
@@ -3,25 +3,30 @@
 # The post path is where the file upload will be posted to.
 # The preview template controls what dropzone looks like(typically dropzone_preview.html found in /layouts)
 # The paramName is the key to the file in the http request dropzone posts.
+# metaFields is a map of json keys in responseData to css selectors the json values should go in
 
 class ELMO.Views.FileUploaderView extends ELMO.Views.ApplicationView
   initialize: (options) ->
     @zoneId = options.zoneId
     @postPath = options.postPath
     @genericThumbPath = options.genericThumbPath
-    @idField = @$('input')
+    @metaFields = options.metaFields
     @previewTemplate = options.previewTemplate
     @paramName = options.paramName
+    @acceptedFileFormats = options.acceptedFileFormats
     @listener = options.listener
 
+
     @dropzone = new Dropzone(@zoneId, {
       url: @postPath
       paramName: @paramName # The name that will be used to transfer the file
       maxFiles: 1
       uploadMultiple: false
       previewTemplate: @previewTemplate,
       thumbnailWidth: 100,
-      thumbnailHeight: 100
+      thumbnailHeight: 100,
+      acceptedFiles: @acceptedFileFormats,
+      maxFilesize: 80000 #kb, fits a maximum row xlsx file
```

Comment:
Curious about this. Strangely, the [docs](https://www.dropzonejs.com/#config-maxFilesize) say this is for limiting the _number_ of files uploaded but that doesn't make sense. 

Elsewhere in the docs there is:

```
// "myAwesomeDropzone" is the camelized version of the HTML element's ID
Dropzone.options.myAwesomeDropzone = {
  paramName: "file", // The name that will be used to transfer the file
  maxFilesize: 2, // MB
  accept: function(file, done) {
    if (file.name == "justinbieber.jpg") {
      done("Naha, you don't.");
    }
    else { done(); }
  }
};
```

which seems to say the unit is MB!?

Was this needed to overcome a failure? Do we know what the default value is?

The nginx server configuration also includes a max upload size restriction. Our default config file currently has it at 50mb.

So we should probably match that.

The thing about size checks is that there are often a crap ton of places where you can specify them and it's often tricky to keep them in sync.

Paperclip has `validates_attachment_size`, but we don't seem to be using that, which is good.

Thoughts?